src/Turbo/src/Twig/TwigExtension.php

```diff
@@ -12,19 +12,24 @@
 namespace Symfony\UX\Turbo\Twig;
 
 use Psr\Container\ContainerInterface;
+use Symfony\Component\HttpFoundation\RequestStack;
+use Symfony\Component\Mercure\Authorization;
 use Symfony\UX\Turbo\Bridge\Mercure\TopicSet;
```

Comment:
I think you could/should probably use TopicSet here, as it's specific to Mercure and would keep things properly typed / isolated.

Or any strategy that does not impact users using a different solution than Mercure.

You should look at this PR and the discussion, as most challenges are identical

https://github.com/symfony/ux/pull/2407

Reply:
I did not mean "use this one", but if you read the PR linked and the discussion, you will see how it relates to yours.

We decided to introduce a new object to bypass these BC questions.